core/tests/org.fusesource.ide.camel.validation.tests.integration/src/org/fusesource/ide/camel/validator/marker/MarkersRemoverValidatorIT.java

```diff
@@ -89,7 +89,7 @@ public void checkMarkers() throws Exception {
 
 		int initial = markers.length;
 		// there are errors or warnings
-		assertThat(initial > 0);
```

Comment:
Optional:
you can write it in an even readable way:
assertThat(markers).hasSize(2);